docs/source/api_reference/tags.rst

```diff
@@ -19,10 +19,15 @@ for this tag.
 This API reference lists all tags available in ``sktime``, and key utilities
 for their usage.
 
+To search estimators by tags on the ``sktime`` webpage, use the
+`Estimator Search Page <https://www.sktime.net/en/latest/estimator_overview.html>`_
```

Comment:
If possible, it'd be better if we can redirect to estimator overview of same version. With this link, it'll always redirect to that corresponding to `main` branch.

Reply:
how would I redirect to same version?

Reply:
genuine question, I do not know how to do that

Reply:
I also never tried myself, but this may work following this [link](https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#the-doc-role).

```suggestion
:doc:`Estimator Search Page </estimator_overview.html>`_
```

Reply:
I applied the suggestion, but keeping the thread open in case it fails.

Reply:
does this work?

Reply:
If I navigate to https://sktime--6816.org.readthedocs.build/en/6816/api_reference/tags.html and click on the hyperlink "Estimator Search Page", it is redirecting me to https://sktime--6816.org.readthedocs.build/en/6816/estimator_overview.html. So, I'll guess that it does work.